<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find sdd_vio)/config/custom_rosconsole.conf"/>
  <node pkg="nodelet"
    type="nodelet"
    name="sdd_vio_node"
    args="standalone sdd_vio/SddVioNodelet"
    output="screen">
    <remap from="~cam_topic_left" to="/cam0/image_raw"/>
    <remap from="~cam_topic_right" to="/cam1/image_raw"/>
    <remap from="~imu_topic" to="/imu0"/>
    <remap from="~cam" to="odom_vio"/>
    <rosparam command="load" file="$(find sdd_vio)/config/vo_param.yaml"/>
    <rosparam command="load" file="$(find sdd_vio)/config/camchain-imucam-euroc.yaml"/>
  </node>
</launch>
